src/pextlib1.0/readline.c

```diff
@@ -309,6 +311,16 @@ int RLHistoryCmd(ClientData clientData UNUSED, Tcl_Interp *interp, int objc, Tcl
 		}
 		s = Tcl_GetString(objv[2]);
 		write_history(s);
+	}  else if (0 == strcmp("append", action)) {
+		if (objc != 3) {
+			Tcl_WrongNumArgs(interp, 1, objv, "append filename");
+			return TCL_ERROR;
+		}
+		s = Tcl_GetString(objv[2]);
+
+		hist_file = fopen(s, "a");
```

Comment:
Can you please add error handling to the `fopen(3)` call here? `mkdtemp.c` has an example on how to do this, e.g.

```c
if (NULL == (hist_file = fopen(s, "a"))) {
    Tcl_AppendResult(interp, "fopen(", s, "): ", strerror(errno), NULL);
    return TCL_ERROR;
}
```